recipes/xgboost/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "xgboost" %}
+{% set version = "0.6a2" %}
+{% set sha256 = "ebc4e2bf8c8266212e342ff8ec4f6ae469e8c41a05d099b6778de8424ce32563" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - Makefile.patch  # [osx]
+    - build-python.patch  # [osx]
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+
+  run:
```

Comment:
One thing I think I forgot to mention is we may want to add `libgcc` for Linux if it is using OpenMP, if the system doesn't provide it. Many systems do, but it could come up if users are running on a minimal setup (e.g. some docker containers). It is easy enough to add if that is the case. So can be saved for the feedstock.